src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -99,14 +100,15 @@ public ResponseEntity<FaqDTO> createFaq(@RequestBody Faq faq, @PathVariable Long
      *         if the faq is not valid or if the faq course id does not match with the path variable
      */
     @PutMapping("courses/{courseId}/faqs/{faqId}")
-    @EnforceAtLeastInstructor
+    @EnforceAtLeastTutorInCourse
     public ResponseEntity<FaqDTO> updateFaq(@RequestBody Faq faq, @PathVariable Long faqId, @PathVariable Long courseId) {
         log.debug("REST request to update Faq : {}", faq);
         if (faqId == null || !faqId.equals(faq.getId())) {
             throw new BadRequestAlertException("Id of FAQ and path must match", ENTITY_NAME, "idNull");
         }
-        authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.INSTRUCTOR, faq.getCourse(), null);
+        checkPriviledgeForAcceptedElseThrow(faq, courseId);
         Faq existingFaq = faqRepository.findByIdElseThrow(faqId);
+        checkPriviledgeForAcceptedElseThrow(existingFaq, courseId);
```

Comment:
Why do we not directly check for `@EnforeAtLeastInstructorInCourse`?

Reply:
Cauuse tutors use the same endpoint to update proposed faq's as well